flink-runtime/src/test/java/org/apache/flink/runtime/state/StateBackendMigrationTestBase.java

```diff
@@ -381,7 +385,7 @@ public void testKeyedMapStateAsIs() throws Exception {
                         stateName, IntSerializer.INSTANCE, new TestType.V1TestTypeSerializer()));
     }
 
-    @Test
+    @TestTemplate
     public void testKeyedMapStateStateMigration() throws Exception {
```

Comment:
```suggestion
    void testKeyedMapStateStateMigration() throws Exception {
```
could be package private